src/core/server/saved_objects/migrations/core/migrate_raw_docs.test.ts

```diff
@@ -58,12 +58,12 @@ describe('migrateRawDocs', () => {
     expect(transform).toHaveBeenNthCalledWith(2, obj2);
   });
 
-  test('passes invalid docs through untouched and logs error', async () => {
+  test('throws when encountering a corrupt saved object document', async () => {
```

Comment:
> When v1 migrations encounter a corrupt saved object it will perform a migration on every restart which can cause data loss in a multi-instance Kibana setup if only one Kibana gets restarted at a time.

Isn't this our recommended way to upgrade kibana? restarting one instance at a time

Reply:
No, all kibana's need to be shutdown before upgrading otherwise v1 migrations will cause data loss.

from https://www.elastic.co/guide/en/kibana/current/upgrade.html
> Shut down all Kibana instances. Running more than one Kibana version against the same Elasticseach index is unsupported. Upgrading while older Kibana instances are running can cause data loss or upgrade failures.